README.md

```diff
@@ -218,6 +218,9 @@ The custom configuration options for the Celery workers are listed below:
   Additionally, it will use this file as a base and set the `overwrite_from_index_token` for the
   registry of the `from_index` container image when applicable. IIB will never directly modify this
   file though. This defaults to `~/.docker/config.json.template`.
+*  `iib_dogpile_backend` - configuration for dogpile.cache backend. Default value is
```

Comment:
Could you please document `iib_dogpile_expiration_time` and `iib_dogpile_arguments` in the readme too?

Reply:
Sure, thanks